AllCommandsOption.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Huttopia\ConsoleBundle;
+
+class AllCommandsOption
+{
+    public static function parseAllCommandsOption(array &$argv): bool
+    {
+        $allCommandsKey = array_search('--all-commands', $argv);
+        if (is_int($allCommandsKey) || is_string($allCommandsKey)) {
+            unset($argv[$allCommandsKey]);
```

Comment:
Maybe test if $allCommandsKey exists?

Reply:
As index is found by array_search on $argv, it should exists ;) i've added a test below for $_SERVER['argv'] to be sure, but it should exists too.